stable/jenkins/README.md

```diff
@@ -61,7 +61,8 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `checkDeprecation`                | Checks for deprecated values used    | `true`                                 |
 | `clusterZone`                     | Override the cluster name for FQDN resolving    | `cluster.local`                |
 | `nameOverride`                    | Override the resource name prefix    | `jenkins`                                 |
-| `fullnameOverride`                | Override the full resource names     | `jenkins-{release-name}` (or `jenkins` if release-name is `jenkins`) |
+| `fullnameOverride`                | Override the deployment namespace    | Not set (`Release.Namespace`)             |
```

Comment:
Explanations for `fullnameOverride` and `namespaceOverride` are mixed up.

Reply:
Thanks for catching this. Resolved!